static/app/views/insights/browser/resources/alerts.ts

```diff
@@ -0,0 +1,22 @@
+import {t} from 'sentry/locale';
+
+export const ALERTS = {
+  spm: {
+    aggregate: 'spm()',
+  },
+  duration: {
+    aggregate: 'avg(d:spans/duration@millisecond)',
+  },
+  decodedSize: {
+    aggregate: 'avg(d:spans/http.decoded_response_content_length@byte)',
```

Comment:
I'm not too sure about this idea, but maybe we should use SpanMetricsField here
```suggestion
    aggregate: `avg(d:spans/${SpanMetricsField.HTTP_DECODED_RESPONSE_CONTENT_LENGTH}@byte)`,
```

The one thing that is stopping me from being confident, is that SpanMetricsField is for discover fields, which is usually the same as the MRI, but not always.